test/Makefile

```diff
@@ -54,6 +54,17 @@ E2EDEV_HOST_IP := $(shell hostname -I | awk '{print $$1}')
 AGBOT_SAPI_URL ?= https://localhost:8083
 ICP_HOST_IP ?= 0
 
+PREBUILT_DOCKER_REG_URL ?= ""
+PREBUILT_DOCKER_REG_USER ?= ""
+PREBUILT_DOCKER_REG_PW ?= ""
+PREBUILT_ANAX_VERSION ?= nightly
+PREBUILT_ESS_VERSION ?= nightly
+export PREBUILT_DOCKER_REG_URL
+export PREBUILT_DOCKER_REG_URL
+export PREBUILT_DOCKER_REG_URL
```

Comment:
Why is this exported 3 times?

Reply:
The exports can also be added to the definition line:
`export PREBUILT_DOCKER_REG_URL ?= ""`